### PR TITLE
Ft-fix-coding-style-errors-157110836

### PR DESCRIPTION
**Pull request by Kalela**
### What does this PR do?
Cleans up the code according to flake8 code standard suggestions.
### How should this be manually tested?
- Clone the repo locally and follow the set up steps in the README.
- cd into the project.
- Set up a virtual environment where you will install your packages.
- If pip is installed, run ```pip install -r requirements.txt```, otherwise, [install pip first](https://pip.pypa.io/en/stable/installing/).
- Run ```pip install flake8``` and run ```flake8``` in the base directory after installation is completed.
- Wait for flake8 to show any errors or warnings in the code.
### What are the relevant pivotal tracker stories?
Python coding style warnings/errors
ID: #157110836

### CODE REVIEW

File: extras/dummy_generator/generator.py [hunk 1]
```diff
@@ -266,7 +279,8 @@
         for i in range(1, args.number_workouts):
 
             uid = str(uuid.uuid4()).split('-')
-            start_date = datetime.date.today() - datetime.timedelta(days=random.randint(0, 100))
+            start_date = datetime.date.today() - \
```
Comment (betsybeth) [inline extras/dummy_generator/generator.py:282]: Good implementation on handling pylint errors

**Review by betsybeth — COMMENTED** [review]
Good work

**Review by FelixWambiri — COMMENTED** [review]
Nice work on implementation.The indentation makes it more readable and easy to follow through.Also the space separating different chunks of code makes it more readable and its nice that you do not have long lines of code.

**Review by meshnesh — APPROVED** [review]
Nice Implementation of the coding styles

**Comment by mutoromike** [comment]
I like how you have fixed previous linting errors. The code is now even much easier to read through.

**Review by frankip — COMMENTED** [review]
good Work, They all follow the PEP8 standards

**Review by FelixWambiri — APPROVED** [review]
Nice work being able to lint the whole application

**Pull request merged**